src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+CockroachDB is made available under the [CockroachDB Software License][csl].
+
+## Types of licenses
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
+
+Type | Description
+-------------|------------
+**Enterprise** <a name="enterprise"></a> | This paid license allows usage of all CockroachDB features with no restrictions under the terms specified in the [CockroachDB Software License][csl]. License must be renewed annually or as negotiated. Support levels available include [Enterprise][support] or [Essential][support].
+**Enterprise Free** <a name="enterprise-free"></a> | Same functionality as **Enterprise**, but free of charge for businesses with less than $10M in annual revenue, and telemetry is required except for ephemeral clusters (7 days or less). Clusters will be [throttled](#throttling) after 7 days without sending telemetry. License must be renewed annually. Support level available is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
+**Enterprise Trial** <a name="enterprise-trial"></a> | A 30 day self-service trial license. Telemetry is required during the trial. Telemetry can be disabled once the cluster is upgraded to a paid **Enterprise** license. Support level available during trials is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
+
+{{site.data.alerts.callout_success}}
+Note that:
+- Clusters with no license have a 7-day grace period before needing to install a license and start sending telemetry data.
```

Comment:
I would remove the "and start sending telemetry data" part. New clusters can use any license key, doesn't have to be a telemetry license.

Reply:
You could also mention that this is useful for ephemeral development clusters.

Reply:
actually - we should think about how to phrase this in docs but: 

Self-hosted clusters without a license key are restricted to development purposes only. Development purposes includes non-production use in an internal development environment for design, prototype, or development purposes only. This restriction is enforced through license terms and conditions.

Reply:
updated this section a bit based on your feedback: (1) remove the "and start sending ..."; (2) mentioned the usefulness for dev clusters; (3) added the unlicensed dev cluster restriction info as an addendum at bottom of section just to make it super clear, also re-ordering the bullets to try to make more sense in context :

---

Note that:
- Clusters with Enterprise Free or Enterprise Trial licenses cannot disable telemetry; if such a cluster signals a telemetry sending error, it will be due to firewall configuration or a network issue.
- Clusters with no license have a 7-day grace period before needing to install a license. This is useful for ephemeral development clusters.
- No license key is required for developers running [single-node clusters](#single-node-clusters).

In general, self-hosted clusters without a license key are restricted to development purposes only. Development purposes includes non-production use in an internal development environment for design, prototype, or development purposes only. This restriction is enforced through [license terms and conditions][csl].